index.html

```diff
@@ -17,25 +17,24 @@
       <div class="logo">
         <a href="#" id="name">Nimrod Acosta</a>
       </div>
-      <button class="normalb">
-        <img src="./Images/Union.png" class="union" alt="hamburger menu" />
-      </button>
-      <nav class="burger">
-        <a href="#">
-          <p class="headermenu">Portfolio</p>
+      <div class="burger" onclick="toggleMenu();">       
+      </div>
+      <nav class="menubar">
```

Comment:
- [ ] Please consider removing the list style for the mobile menu links, so your design is cleaner and as close as possible to the Figma design
- [ ] [OPTIONAL] You can give the mobile menu a height of `100vh` and width of `100vw`, this will hide everything else will in the menu

|Your design|Figma design|
|-|-|
|![Screenshot_20220614_191341](https://user-images.githubusercontent.com/25227022/173660897-46f52555-37a0-4106-a962-525e1c07926f.png)|![Screenshot_20220614_192252](https://user-images.githubusercontent.com/25227022/173661440-faa08857-2fa2-4f53-ba39-0506a5794c3c.png)|